config/locales/server.en.yml

```diff
@@ -8,6 +8,7 @@ en:
     solved_topics_auto_close_hours: "Auto close topic (n) hours after the last reply once the topic has been marked as solved. Set to 0 to disable auto closing."
     show_filter_by_solved_status: "Show a dropdown to filter a topic list by solved status."
     notify_on_staff_accept_solved: "Send notification to the topic creator when a post is marked as solution by a staff."
+    disable_solved_education_message: "Disable education message for solved topics."
```

Comment:
Do we need a setting for this? Has anyone asked for it, or do we plan to remove it at some point?

Reply:
@SamSaffron asked for a setting.